### PR TITLE
chore: add device model and replace device name

### PR DESCRIPTION
**Pull request by marandaneto**
## Problem

Noticed by checking the `most common devices` dashboard template that RN used the wrong property for device model and name

### Bump level

- [ ] Major
- [ ] Minor
- [x] Patch

### Libraries affected

- [ ] All of them
- [ ] posthog-web
- [ ] posthog-node
- [x] posthog-react-native

### Changelog notes

- Added support for X

### CODE REVIEW

**Review by benjackwhite — COMMENTED** [review]
Code-wise no issues, but is this technically a breaking change? Not so much for the SDK but for the analytics data on the other end 🤔

File: posthog-react-native/src/native-deps.tsx [hunk 1]
```diff
@@ -40,12 +41,13 @@ export const getAppProperties = (): PostHogCustomAppProperties => {
   }
 
   if (OptionalReactNativeDeviceInfo) {
-    properties.$app_build = returnPropertyIfNotUnknown(OptionalReactNativeDeviceInfo.getBuildIdSync())
+    properties.$app_build = returnPropertyIfNotUnknown(OptionalReactNativeDeviceInfo.getBuildNumber())
```
Comment (marandaneto) [inline posthog-react-native/src/native-deps.tsx:44]: This matches the right app build when using expo-device and the Android/iOS SDKs
getBuildId != BuildNumber

**Review by benjackwhite — APPROVED** [review]

**Comment by marandaneto** [comment]
the RN build command is not packing the `uuidv7` dep so I have to fix that first.

File: posthog-react-native/src/native-deps.tsx (hunk 1 above)
Comment (marandaneto) [reply posthog-react-native/src/native-deps.tsx:44]: it was returning `UP1A.231105.003` instead of `1` which is the app build

**Comment by marandaneto** [comment]
close in favor of https://github.com/PostHog/posthog-js-lite/pull/185

**Pull request closed without merging**